lxd/daemon.go

```diff
@@ -549,37 +553,35 @@ func (d *Daemon) init() error {
 	if err != nil {
 		return err
 	}
+
 	err = d.setupExternalAuthentication(macaroonEndpoint)
 	if err != nil {
 		return err
 	}
+
 	if !d.os.MockMode {
-		/* Start the scheduler */
+		// Start the scheduler
 		go deviceEventListener(d.State())
 
+		// Setup inotify watches
 		_, err := deviceInotifyInit(d.State())
 		if err != nil {
 			return err
 		}
 
 		deviceInotifyDirRescan(d.State())
-
 		go deviceInotifyHandler(d.State())
 
+		// Read the trusted certificates
 		readSavedClientCAList(d)
 	}
 
+	// Connect to MAAS
 	err = d.setupMAASController(maasAPIURL, maasAPIKey, maasMachine)
 	if err != nil {
 		return err
 	}
 
-	if !d.os.MockMode {
-		/* Start the scheduler */
-		go deviceEventListener(d.State())
-		readSavedClientCAList(d)
-	}
```

Comment:
Huh, when did that get in.

Reply:
Probably a bad merge in the clustering branch.